mantidproject/mantid
# Indirect - ConvFit - Enable Plot Guess for Diffusion Functions

Currently, in the ConvFit interface, the plot guess feature is disable for the Diffusion functions.

This PR enables plot guess and ensure it works as intended for all functions in ConvFit.

**To test:**
1. Navigate to Interfaces > Indirect > Data Analysis > ConvFit.
2. Supply the provided \*\_red file into the Sample field.
3. Supply the provided \*\_res file into the Resolution field.
4. For each fit function in the Fit Type drop-down menu:
    1. Select the fit type in the Fit Type drop-down menu.
    2.
        a) OPTIONAL (to get reasonable fit parameters) Click 'Run'
        b) Use plot guess to create a guess plot with the fit parameters.
    3. Ensure the guess is plotted (green plot) in the top preview plot (if 'Run' was clicked, guess plot should look very similar to fit plot).
    4. Move the blue fit selected in the top preview plot and ensure the guess plot updates accordingly.

**Must be merged after PR #20933**

**Test Files:**
[convfit-testfiles.zip](https://github.com/mantidproject/mantid/files/1401655/convfit-testfiles.zip)



Fixes #13345. 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
